lisp/bazel-build.el

```diff
@@ -24,38 +24,56 @@
 
 ;;; Code:
 
+(require 'subr-x)
+
 (require 'bazel-util)
 
+(defgroup bazel-build nil
+  "Package for editing, building, and running code using Bazel."
+  :link '(url-link "https://bazel.build")
+  :link '(url-link "https://github.com/bazelbuild/emacs-bazel-mode")
+  :group 'languages)
+
+(defcustom bazel-build-bazel-command '("bazel")
+  "Command and arguments that should be used to invoke Bazel."
+  :type '(repeat string)
+  :risky t
+  :group 'bazel-build)
+
 (defun bazel-build (target)
   "Build a Bazel TARGET."
-  (interactive (list (bazel-build--read-target "bazel build ")))
+  (interactive (list (bazel-build--read-target "build")))
   (bazel-build--run-bazel-command "build" target))
 
 (defun bazel-run (target)
   "Build and run a Bazel TARGET."
-  (interactive (list (bazel-build--read-target "bazel run ")))
+  (interactive (list (bazel-build--read-target "run")))
   (bazel-build--run-bazel-command "run" target))
 
 (defun bazel-test (target)
   "Build and run a Bazel test TARGET."
-  (interactive (list (bazel-build--read-target "bazel test ")))
+  (interactive (list (bazel-build--read-target "test")))
   (bazel-build--run-bazel-command "test" target))
 
 (defun bazel-build--run-bazel-command (command target)
   "Run Bazel tool with given COMMAND, e.g. build or run, on the given TARGET."
   (compile
-   (mapconcat #'shell-quote-argument (list "bazel" command target) " ")))
+   (mapconcat #'shell-quote-argument
+              (append bazel-build-bazel-command (list command target)) " ")))
 
-(defun bazel-build--read-target (prompt)
-  "Read a Bazel build target from the minibuffer.  PROMPT is a read-only prompt."
+(defun bazel-build--read-target (command)
+  "Read a Bazel build target from the minibuffer.
+COMMAND is a read-only Bazel command."
```

Comment:
"read-only" comes from the prompt argument before, now it's confusing (why should it matter that the Bazel command is read-only?). I'd say something like "COMMAND is a Bazel command to be included in the minibuffer prompt".

Reply:
Agreed. Fixed.